### PR TITLE
Removes stream observers once session is disconnected [iOS]

### PR DESCRIPTION
**Pull request by amatakasap**
[#94](https://github.com/opentok/cordova-plugin-opentok/issues/94)

### CODE REVIEW

File: src/ios/OpenTokPlugin.m [hunk 1]
```diff
@@ -610,6 +610,10 @@ - (void)sessionDidDisconnect:(OTSession*)session{
     if( _publisher ){
         [_publisher.view removeFromSuperview];
     }
+    // Remove session observers
+    for ( id key in streamDictionary ) {
+        [self removeObserversFromStream: [streamDictionary objectForKey:key]];
+    }
```
Comment (wolfenrain) [inline src/ios/OpenTokPlugin.m]: We should maybe also remove all objects within the streamDictionary then for memory sake(like a few lines above) `[streamDictionary removeAllObjects]`.

Comment (msach22) [reply src/ios/OpenTokPlugin.m]: I agree that we should free up resources and remove all objects. @amatakasap can you please update the PR?

**Comment by amatakasap** [comment]
I'm sorry. I do not know how to do it. What should I do?

**Comment by msach22** [comment]
@amatakasap I just updated your PR with the code that we needed 😄

**Comment by amatakasap** [comment]
Thank you.

**Pull request merged**